src/test/cpp/throughput/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+option(BUILD_THROUGHPUT_TESTS "Build throughput tests"  OFF)
+
+if( NOT BUILD_THROUGHPUT_TESTS )
```

Comment:
Also exit with a message if the FMT library was found

Reply:
Since it's marked as required, it will do that already.